src/Storage/Controllers/InstancesController.cs

```diff
@@ -141,51 +141,53 @@ public class InstancesController : ControllerBase
                 return BadRequest("Both InstanceOwner.PartyId and InstanceOwnerIdentifier cannot be present at the same time.");
             }
 
-            if (orgClaim != null)
+            switch ((orgClaim != null, userId != null))
```

Comment:
I think the if statements are easier to make sense of compared to the switch. What motivated the change?